loader/include/Geode/loader/Mod.hpp

```diff
@@ -79,9 +80,13 @@ namespace geode {
         bool isEnabled() const;
         bool isLoaded() const;
         bool supportsDisabling() const;
-        bool supportsUnloading() const;
-        bool wasSuccesfullyLoaded() const;
-        ModInfo getModInfo() const;
+        bool canDisable() const;
+        bool canEnable() const;
+        [[deprecated]] bool supportsUnloading() const;
+        [[deprecated("wasSuccessfullyLoaded")]] bool wasSuccesfullyLoaded() const;
```

Comment:
what is this deprecated message for

Reply:
typo (wasSucceSfullyLoaded vs wasSucceSSfullyLoaded)

Reply:
make it more clear imo